motr/io_pargrp.c

```diff
@@ -2505,8 +2506,8 @@ M0_INTERNAL void pargrp_iomap_fini(struct pargrp_iomap *map,
 	m0_indexvec_free(&map->pi_ivec);
 
 	for (row = 0; row < rows_nr(play, obj); ++row) {
-		if (map->pi_ioo->ioo_pbuf_type == M0_PBUF_IND &&
-		    map->pi_databufs[row][0] == NULL) {
+		if (map->pi_state == PI_DEGRADED &&
```

Comment:
Won't this impact happy path when write case or read verify mode ?

Reply:
Should not impact to write path, but for read-verify might impact let me check that.

Reply:
Even in case of write, read verify I don't see parity_buf_type is set to M0_PBUF_IND (Except, case of replication (N=1) for writes)and also the map->pi_databufs[row][0] was never NULL in any cases. Probably this check was wrong earlier. 
Don't see it is impacting write or read verify path with this fix, as with "alloc-profile" don't show any leaks with/without fix.

Reply:
Removed this change now.